.hashibot.hcl

```diff
@@ -34,3 +34,16 @@ poll "label_issue_migrater" "provider_migrater" {
     EOF
     migrated_comment = "This issue has been automatically migrated to ${var.repository}#${var.issue_number} because it looks like an issue with that provider. If you believe this is _not_ an issue with the provider, please reply to ${var.repository}#${var.issue_number}."
 }
+
+poll "closed_issue_locker" "locker" {
+  schedule             = "0 50 1 * * *"
+  closed_for           = "720h" # 30 days
+  max_issues           = 500
+  sleep_between_issues = "5s"
+
+  message = <<-EOF
+    I'm going to lock this issue because it has been closed for _30 days_ ⏳. This helps our maintainers find and focus on the active issues.
+
+    If you feel this issue should be reopened, we encourage creating a new issue linking back to this one for added context. If you feel I made an error 🤖 🙉, please reach out to my human friends 👉 hashibot-feedback@hashicorp.com. Thanks!
```

Comment:
I'd prefer to see this encourage opening a new issue rather than adding another queue into the mix.

Reply:
It does in fact encourage opening a new issue, but I will change the sentence not even mention the words "reopen"